initDB.sql

```diff
@@ -0,0 +1,13 @@
+create schema if not exists department; 
+create table if not exists department.employee
+(
+    id serial primary key,
```

Comment:
I understand that there is no clear technical specification, but think about what constraints you can add to these fields, now you can add a completely empty record to your table